features/filiatie.feature

```diff
@@ -0,0 +1,618 @@
+            Als "/kadastraalonroerendezaken/A9" wordt gevraagd
+            Dan bevat het antwoord "isOntstaanUit" met waarde:    
+                """
+                [
+                    {
+                        "kadastraalOnroerendeZaak": {
+                            "identificatie": "P12"
+                        }
+                    }
+                ]
+                """
+            En bevat het antwoord "bijbehorendeGrondperceelIdentificaties": [ "P12" ]
+            En bevat het antwoord in "bijbehorendeGrondperceelIdentificaties" geen waarde "P11"
+            En bevat het antwoord in "bijbehorendeGrondperceelIdentificaties" geen waarde "P13"
+            En bevat het antwoord "volgnummer": 1
+            En bevat het antwoord geen property "indicatieVervallen"
+            En bevat het antwoord geen property "einddatum"
+            En is de kadastraal onroerende zaak niet sluimerend
+            En bevat het antwoord geen property "isOvergegaanIn"
+            En bevat het antwoord geen property "bijbehorendeAppartementsrechten"
+
+        Scenario: Opvragen appartementsrecht met grondperceel uit vereniging
+            Gegeven Percelen met identificatie "P14" en "P15" zijn verenigd naar perceel "P16"
+            En Perceel met identificatie "P16" is gesplitst naar appartementsrechten "A11" en "A12"
+            En Appartementsrecht met identificatie "A11" is niet verder gesplitst of verenigd
+            En Appartementsrecht met identificatie "A11" heeft "volgnummer"= 1
+            En Appartementsrecht met identificatie "A12" is niet verder gesplitst of verenigd
+            En Appartementsrecht met identificatie "A12" heeft "volgnummer"= 2
+            Als "/kadastraalonroerendezaken/A11" wordt gevraagd
+            Dan bevat het antwoord "isOntstaanUit" met waarde:    
+                """
+                [
+                    {
+                        "kadastraalOnroerendeZaak": {
+                            "identificatie": "P16"
+                        }
+                    }
+                ]
+                """
+            En bevat het antwoord "bijbehorendeGrondperceelIdentificaties": [ "P16" ]
+            En bevat het antwoord in "bijbehorendeGrondperceelIdentificaties" geen waarde "P14"
+            En bevat het antwoord in "bijbehorendeGrondperceelIdentificaties" geen waarde "P15"
+            En bevat het antwoord "volgnummer": 1
+            En bevat het antwoord geen property "indicatieVervallen"
+            En bevat het antwoord geen property "einddatum"
+            En is de kadastraal onroerende zaak niet sluimerend
+            En bevat het antwoord geen property "isOvergegaanIn"
+            En bevat het antwoord geen property "bijbehorendeAppartementsrechten"
+    
+    Rule: Na een wijzigingssplitsing van een appartementsrecht worden de vervallen appartementsrechten niet getoond en wordt de filiatie uit het oude appartementsrecht overgenomen in de nieuwe appartementsrechten
```

Comment:
Er worden altijd per appartementsrecht unieke volgnummers gecreëerd. Ook bij wijzigingssplitsing.  Dus in dit geval zou A18 volgnummer 6 hebben, A19 volgnummer 7 & A20 volgnummer 8.

Reply:
dank. copy-paste foutje. opgelost